### DISCUSSION TITLE
Storage and displaying of graphs

### DISCUSSION THREAD

**Discussion by subaangi06** [YetAnotherPomodoroApp/YAPA-2#111] [Ideas]
Love the app, use it to mainly track how long I've studied for, I think it'll be cool for me to look back on it when I'm older. Thought that it'll be cool if we could store data from the app and display it as graphs externally, maybe on a website or something similar. In my knowledge, there is no method of storing data, and Im pretty sure that data will get lost if the app gets deleted, or we switch to a new device. Hope this idea of exporting data makes sense

**Comment by floatas** [comment]
Hello,
Thank you for the suggestion. I agree that there should be some sort of open export, something like CSV might be the best solution since you could import to Excel or any other tool and draw charts.
